README.md

```diff
@@ -422,6 +424,64 @@ w. (0T1)d2(0T1)d4
 b. (0T1)d7(0T1)d6
 ```
 
+## Hashing
+
+For certain applications, getting a hash or checksum of either the full state (all boards) or a single board is desired. To facilitate standardization, hashing both the full state and a single board should be create by the following processes.
+
+*This standard is still just a proposal and not official yet!*
+
+For single board hashing:
+
+  1. Create the 5DFEN string of the single board, stripping any and all whitespaces (including newlines)
+  2. Feed the resulting string (encoded as UTF-8) into the MD5 message-digest algorithm
+  3. Extract the resulting hash as hexadecimal digits
+  4. Convert to UTF-8 string (alphabet characters should be lowercase)
+
+For full state hashing:
+
+  1. Grab the 5DFEN string of all boards, sorting boards by timeline (ascending) and then by turn (ascending)
+  2. Concatenate all of these 5DFEN strings together (keeping the order), stripping any and all whitespaces (including newlines)
+  3. Feed the resulting string (encoded as UTF-8) into the MD5 message-digest algorithm
+  4. Extract the resulting hash as hexadecimal digits
+  5. Convert to UTF-8 string (alphabet characters should be lowercase)
```

Comment:
*Note:* this is equivalent to keeping the hash state and hashing each board in the specified order

Reply:
Not sure what you mean by this

Reply:
It's just another way to compute the hash, that only requires O(1) memory